demo/README.md

```diff
@@ -0,0 +1,33 @@
+# Demo Application
+
+**Warning**: This is just a simple demo setup that is not suited for production use!
+
+## Prerequisites
+
+1. Get your mashine's IP address
+1. Replace `IP_PLACEHOLDER` with your IP in `.env`.
```

Comment:
minor: clear to me, but below the path `./demo` is given, which is missing here.